tests/ci/cdk/cdk/ssm/m1_tests_ssm_document.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+---
+schemaVersion: '2.2'
+description: aws-lc:bmFrameworkEc2Benchmark
+mainSteps:
+  - action: aws:runShellScript
+    name: runShellScript
+    inputs:
+      timeoutSeconds: '7200'
+      runCommand:
+        - eval $(/opt/homebrew/bin/brew shellenv)
+        - sudo su
+        - cd /tmp
```

Comment:
Can this handle multiple builds in parallel if it creates a unique test directory per test? Perhaps with [mktemp](https://linux.die.net/man/1/mktemp)